src/Dfe.PlanTech.Web/ProgramExtensions.cs

```diff
@@ -133,7 +133,7 @@ public static IServiceCollection AddDatabase(this IServiceCollection services, I
 
         services.AddEFSecondLevelCache(options =>
         {
-            options.UseMemoryCacheProvider().DisableLogging(false).UseCacheKeyPrefix("EF_");
+            options.UseMemoryCacheProvider().ConfigureLogging(false).UseCacheKeyPrefix("EF_");
```

Comment:
Shouldn't this be `ConfigureLogging(true)` (or something) to match the original intent?

Reply:
Well the original was saying don't disable logging, and defaults to our appsettings, whereas the new function if set to true overwrites your defaults and sets the level of logging to 'Debug' if I'm reading the docs correctly...ours are set to warning